### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by Sophie-Xie**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### What does this PR do?
In order to review more efficiently, update the PR template.

#### Which issue(s)/PR(s) this PR relates to?
N/A
  
#### Special notes for your reviewer, ex. impact of this fix, etc:
N/A

#### Additional context/ Design document:
N/A

#### Checklist:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> None                                                                `

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,27 +1,42 @@
...
+- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
 - [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
 - [ ] Performance impacted: Consumes more CPU/Memory
 
+
 #### Release notes:
 
 Please confirm whether to be reflected in release notes and how to describe:
->                                                                 `
+>                                                                   `
```
Comment (wey-gu) [inline .github/pull_request_template.md:42]: Shouldn't we leave the spaces and `` ` of this line, considering we are changing it in this chance :)?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,27 +1,42 @@
...
 
+  
+#### Special notes for your reviewer, ex. impact of this fix, design document, etc:
 
-#### Additional context/ Design document:
 
 
 #### Checklist:
+Tests:
+- [ ] Unit test(normal and abnormal processes)
```
Comment (wey-gu) [inline .github/pull_request_template.md:27]: (positive and negative cases)

**Review by wey-gu — COMMENTED** [review]
LGTM, thanks!
some minor wording commented inline.

**Review by wey-gu — APPROVED** [review]
LGTM, thanks!

**Review by Aiee — APPROVED** [review]

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,27 +1,42 @@
...
+#### Special notes for your reviewer, ex. impact of this fix, design document, etc:
 
-#### Additional context/ Design document:
 
 
 #### Checklist:
+Tests:
+- [ ] Unit test(positive and negative cases)
+- [ ] Function test
+- [ ] Perfomance test
```
Comment (Shylock-Hg) [inline .github/pull_request_template.md:29]: Now we don't have performance test in ci.

Comment (Sophie-Xie) [reply .github/pull_request_template.md:29]: emmm, but i think we need test it by others.

Comment (Shylock-Hg) [reply .github/pull_request_template.md:29]: Yes, in my mind, I think we could add it to CI or a lightweight performance test.

Comment (Sophie-Xie) [reply .github/pull_request_template.md:29]: Will do it in test platform.

**Review by yixinglu — APPROVED** [review]

**Pull request merged**